spec/dummy/spec/features/integration_spec.rb

```diff
@@ -176,6 +176,15 @@ def change_text_expect_dom_selector(dom_selector)
   end
 end
 
+feature "renderedHtml from generator function", :js do
+  subject { page }
+  background { visit "/rendered_html" }
+  scenario "renderedHtml should not have any errors" do
+    expect(subject).to have_text "Props: {\"hello\":\"world\"}"
+    expect(subject.html).to include("[SERVER] RENDERED RenderedHtml to dom node with id")
```

Comment:
Why do we have to check with the whole string here instead of putting assertion on what we want to verify each part.